src/api/app/components/card_component.html.haml

```diff
@@ -0,0 +1,17 @@
+.col-12.col-md-6.col-lg-4
+  .card.mb-3.repository-card
+    .card-header.d-flex.justify-content-between
+      %div
+        = title
+        - tags.each do |tag|
```

Comment:
This is specific to how the view component would be used in the context of repositories. It doesn't apply to other contexts. You should rely on [slots](https://viewcomponent.org/guide/slots.html). You could have a slot for the card header, body and footer for example.

Reply:
Oh, nice tip... We'll take a look at slots for sure